Hw 3

ivirna opened the pull request.

samodurOFF commented on HW_3/task1.py:
```diff
@@ -0,0 +1,19 @@
+# чисел Ai. Последняя строка содержит число x
+
+
+print('Введите натуральное число N') 
+N = abs(int(input()))
```
Без функции `print` здесь можно вполне обойтись.
```suggestion
N = abs(int(input('Введите натуральное число N '))
```

samodurOFF commented on HW_3/task1.py:
```diff
@@ -0,0 +1,19 @@
+else:
+    x = int(input('Введите число X, которое необходимо найти в списке: '))
+    count = 0
+    for i in range(N):
```
Можно приходиться напрямую по элементам списка.

samodurOFF commented on HW_3/task2.py:
```diff
@@ -0,0 +1,22 @@
+    x = int(input('Введите число X, с которым необходимо сравнивать элементы списка: '))
+    min = abs(x - A_num[0])
+    index = 0
+    for i in range(1, N):
```
И тут тоже

ivirna commented:
Комментарии понятны. Большое спасибо и хорошего дня! 
    вторник, 18 апреля 2023 г. в 10:06:41 GMT+3, Danil ***@***.***> написал(-а):  
 
 


@samodurOFF commented on this pull request.

In HW_3/task1.py:
> +print('Введите натуральное число N') 
+N = abs(int(input()))

Без функции print здесь можно вполне обойтись.
⬇️ Suggested change-print('Введите натуральное число N') 
-N = abs(int(input()))
+N = abs(int(input('Введите натуральное число N '))

In HW_3/task1.py:
> +# Задача 16: Требуется вычислить, сколько раз встречается некоторое число Х
+# в массиве А[1..N]. Пользователь в первой строке вводит натуральное число N -
+# количество элементов в массиве. В последующих строках записаны N целых
+# чисел Ai. Последняя строка содержит число x
+
+
+print('Введите натуральное число N') 
+N = abs(int(input()))
+A_entered = input('Введите через пробел элементы списка: ').split()
+A_num = list(map(int, A_entered))
+if len(A_num) != N:
+    print('Введенные элементы не соответствуют заявленному количеству!')
+else:
+    x = int(input('Введите число X, которое необходимо найти в списке: '))
+    count = 0
+    for i in range(N):

Можно приходиться напрямую по элементам списка.

In HW_3/task2.py:
> +# к заданному числу X. Пользователь в первой строке вводит натуральное число N -
+# количество элементов в массиве. В последующих строках записаны N целых чисел Ai.
+# Последняя строка содержит число X 
+
+
+print('Введите натуральное число N') 
+N = abs(int(input()))
+A_entered = input('Введите через пробел элементы списка: ').split()
+A_num = list(map(int, A_entered))
+if len(A_num) != N or N == 0:
+    print('Введенные элементы не соответствуют заявленному количеству!')
+else:
+    x = int(input('Введите число X, с которым необходимо сравнивать элементы списка: '))
+    min = abs(x - A_num[0])
+    index = 0
+    for i in range(1, N):

И тут тоже

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>
